src/formats/templates/compose/extra-colors.kt.template

```diff
@@ -24,10 +11,23 @@ import androidx.compose.ui.graphics.Color
 <%= options.import.map(function(item) {
     return 'import ' + item
   }).join('\n') %>
+
+<%= fileHeader({file, commentStyle: 'short'}) %>
+
+<%
+function colorName(prop) {
+  let name = prop.name.replace(/^color/, "");
+  return name.charAt(0).toLowerCase() + name.slice(1);
+}
```

Comment:
I believe this roughly does what `camelCaseDecimal.ts` do. Could we try to piggy back off that function rather than defining a new one here?

Reply:
Moved this to a utils function, although I still need to do some mapping of these values in the template (calling this function, storing the result).